pkg/storage/my_requests.go

```diff
@@ -4,25 +4,55 @@ import (
 	"context"
 	"database/sql"
 	"errors"
+	"time"
 
+	"github.com/google/uuid"
+	"github.com/guregu/null"
 	"go.uber.org/zap"
 
 	"github.com/cmsgov/easi-app/pkg/appcontext"
 	"github.com/cmsgov/easi-app/pkg/apperrors"
-	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/graph/model"
 )
 
+// Request defines the generic request object pulled from multiple db tables
+type Request struct {
+	ID          uuid.UUID
+	Name        null.String
+	SubmittedAt *time.Time        `db:"submitted_at"`
+	CreatedAt   time.Time         `db:"created_at"`
+	Type        model.RequestType `db:"record_type"`
+}
+
 // FetchMyRequests queries the DB for an accessibility requests.
 // TODO implement cursor pagination
-func (s *Store) FetchMyRequests(ctx context.Context) ([]models.AccessibilityRequest, error) {
+func (s *Store) FetchMyRequests(ctx context.Context) ([]Request, error) {
 	principal := appcontext.Principal(ctx)
 	EUAUserID := principal.ID()
 
-	requests := []models.AccessibilityRequest{}
+	requests := []Request{}
 
-	requestsSQL := `SELECT * FROM accessibility_requests
-		WHERE deleted_at IS NULL
-		AND eua_user_id = $1
+	requestsSQL := `
+		SELECT 
+			id,
+			name,
+			created_at AS submitted_at,
+			'ACCESSIBILITY_REQUEST' record_type,
+			created_at
+		FROM accessibility_requests
+			WHERE deleted_at IS NULL
+			AND eua_user_id = $1
```

Comment:
Do we have an index on these columns? If not we should probably add a story to add one. Same with the second part of the query below for the other table.

Reply:
Good call, Jim! I will do that in some follow-up work.